parser/mysql/type.go

```diff
@@ -74,6 +74,7 @@ const (
 	PreventNullInsertFlag uint = 1 << 20 /* Prevent this Field from inserting NULL values */
 	EnumSetAsIntFlag      uint = 1 << 21 /* Internal: Used for inferring enum eval type. */
 	DropColumnIndexFlag   uint = 1 << 22 /* Internal: Used for indicate the column is being dropped with index */
+	StringToDecimalFlag   uint = 1 << 23 /* Internal: Used for indicate string to decimal */
```

Comment:
Looks like it's not used

Reply:
FixUnixtimePrecision used in this function